packages/rich-text/src/component/index.js

```diff
@@ -300,6 +301,12 @@ class RichText extends Component {
 		this.props.onSelectionChange( index, index );
 		this.setState( { activeFormats } );
 
+		// Update selection as soon as possible, which is at the next animation
+		// frame. The event listener for selection changes may be added too late
+		// at this point, but this focus event is still too early to calcutate
+		// the selection.
```

Comment:
Small typo: “calculate”

Reply:
Fixed in 9cb57e40d41adfc203aaa089926b5d6822744beb 👍